README.md

```diff
@@ -80,15 +80,22 @@ Chitragupta.setupProcessLogger('processing_crazy_stuff', process_some_crazy_stuf
 
 For workers
 ```node
-const getWorkFromRedis = (n, first_run) =>
+const processFromQueue = (n) => {
```

Comment:
not using the parameter **n** anywhere?

Reply:
Just an example, should not make a difference :)